test/lint/licenses.sh

```diff
@@ -1,2 +1,3 @@
 #!/bin/sh -eu
+shell command -v go-licenses || go install github.com/google/go-licenses@latest
```

Comment:
The `go install` call is already in the Makefile alongside the other checks used for static analysis

Reply:
done